### PR TITLE
dm: optimize query status doc

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/dm-query-status.md [hunk 1]
```diff
@@ -206,24 +202,64 @@ The status of a DM migration task depends on the status of each subtask assigned
...
+            - `progress`: The progress of the loading process.
+            - `bps`: The speed of the full loading.
+        - `dump`: The replication information of the `Dump` processing unit.
+            - `totalTables`: The number of tables to be dumped.
+            - `completedTables`:  The number of tables that have been dumped.
+            - `finishedBytes`: The number of bytes that have been dumped.
+            - `finishedRows`: The number of rows that have been dumped.
+            - `estimateTotalRows`: The estimated number of rows to be dumped.
+            - `progress`: The progress of the dumping process.
+            - `bps`: The dumping speed.
```
Comment (dveeden) [inline dm/dm-query-status.md:262]:
```suggestion
            - `bps`: The dumping speed in bytes/second.
```

**Review by dveeden — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: dm/dm-query-status.md [hunk 2]
```diff
@@ -206,24 +202,64 @@ The status of a DM migration task depends on the status of each subtask assigned
...
+- `sources`: The list of upstream MySQL instances. Each task contains the following fields:
+    - `result`
+    - `msg`
+    - `sourceStatus`: The information of the upstream MySQL databases.
+    - `subTaskStatus`: The information of all subtasks of upstream MySQL databases. Each subtask might contain the following fields:
+        - `name`: The name of the subtask.
+        - `stage`: The status of the subtask. For the status description and status switch relationship of "stage" of "subTaskStatus" of "sources", see the [subtask status](#subtask-status).
+        - `unit`: The processing unit of DM, including "Check", "Dump", "Load", and "Sync".
+        - `result`: Displays the error information if a subtask fails.
+        - `unresolvedDDLLockID`: The sharding DDL lock ID, used for manually handling the sharding DDL lock in the abnormal condition.
```
Comment (ran-huang) [inline dm/dm-query-status.md:234]:
```suggestion
        - `unresolvedDDLLockID`: The sharding DDL lock ID, used for manually handling the sharding DDL lock in the abnormal condition. For operation details of "unresolvedDDLLockID" of "subTaskStatus" of "sources", see [Handle Sharding DDL Locks Manually](/dm/manually-handling-sharding-ddl-locks.md).
```

File: dm/dm-query-status.md [hunk 3]
```diff
@@ -206,24 +202,64 @@ The status of a DM migration task depends on the status of each subtask assigned
...
+        - `dump`: The replication information of the `Dump` processing unit.
+            - `totalTables`: The number of tables to be dumped.
+            - `completedTables`:  The number of tables that have been dumped.
+            - `finishedBytes`: The number of bytes that have been dumped.
+            - `finishedRows`: The number of rows that have been dumped.
+            - `estimateTotalRows`: The estimated number of rows to be dumped.
+            - `progress`: The progress of the dumping process.
+            - `bps`: The dumping speed in bytes/second.
 
 For operation details of "unresolvedDDLLockID" of "subTaskStatus" of "sources", see [Handle Sharding DDL Locks Manually](/dm/manually-handling-sharding-ddl-locks.md).
```
Comment (ran-huang) [inline dm/dm-query-status.md:264]:
```suggestion
```

[the remark this answers is not in the document]
Comment (ran-huang) [reply dm/dm-query-status.md]: done

**Pull request merged**